lib/application.js

```diff
@@ -182,7 +195,7 @@ app.handle = function handle(req, res, callback) {
  * See Router#use() documentation for details.
  *
  * If the _fn_ parameter is an express app, then it will be
- * mounted at the _route_ specified.
+ * mounted at the _route_ specifiedy
```

Comment:
A typo appeared here after rebase from 9f34d06cf85cb7f5542342db895c066f6869b192 to 77ada8512a903b8e27b52b5caf649e5cc464125e

```suggestion
 * mounted at the _route_ specified.
```